Makefile

```diff
@@ -20,7 +20,7 @@ SHELL:=/usr/bin/env bash
 
 .DEFAULT_GOAL:=help
 
-GO_VERSION ?= 1.20.4
+GO_VERSION ?= 1.20.8
```

Comment:
Same quesiton as in IPAM. This is not bumping Go.

Reply:
I have updated the docker file, let me know if I have missed somewhere else. I will update ipam PRs too.